src/endpoints/mex/mex.pair.service.ts

```diff
@@ -128,10 +137,22 @@ export class MexPairService {
     const secondTokenSymbol = pair.secondToken.identifier.split('-')[0];
     const state = this.getPairState(pair.state);
     const type = this.getPairType(pair.type);
+
     if (!type || [MexPairType.unlisted].includes(type)) {
       return undefined;
     }
 
+    let exchange: MexPairExchange;
+
+    if (type === MexPairType.core || type === MexPairType.community ||
```

Comment:
you can use the includes function on an array to make this more compact

Reply:
done